app/src/test/kotlin/fr/free/nrw/commons/upload/UploadClientTest.kt

```diff
@@ -162,15 +165,23 @@ class UploadClientTest {
         whenever(contribution.fileKey).thenReturn(filekey)
         whenever(fileUtilsWrapper.getMimeType(anyOrNull<File>())).thenReturn("image/png")
         whenever(fileUtilsWrapper.getFileChunks(anyOrNull<File>(), eq(expectedChunkSize))).thenReturn(emptyList())
-
         val result = uploadClient.uploadFileToStash(filename, contribution, mock()).test()
-
         result.assertNoErrors()
-        verify(contribution, times(1)).unpause()
+        verify(contribution, times(1))
     }
 
     @Test
     fun uploadFileToStash_returnsFailureIfNothingToUpload() {
+//        val tempFile = File.createTempFile("tempFile", ".tmp")
```

Comment:
Is this test still required?

Reply:
Should we remove it since we are not working with if nothing is to upload.

Reply:
Could you point to the functionality that this test was written for? Has it been removed in this pull request or did it never exist? If we're no longer using that functionality, then we can remove it.